### DISCUSSION TITLE
1.6.1

### DISCUSSION THREAD

**Discussion by jojo2357** [jojo2357/Music-Discord-Rich-Presence#49] [General]
This release fixes an unreported minor bug that caused the console to not reset properly when using MusicBee (no effect on DRP), and this release also allows more customization with the `Title` and `Artist` lines.

Before: 
![image](https://user-images.githubusercontent.com/66704796/140189512-bb496ae6-d133-48b3-bb69-db28db158b4e.png)

After (mimicking Spotify's presence): 
![image](https://user-images.githubusercontent.com/66704796/140189740-9d9d0708-1a06-404e-8a05-f9866cb1ac2b.png)

Now you can do almost anything you want there, and the ini contains the template for both styles so just remove the one you dont want (and also remove the `//`)

<hr /><em>This discussion was created from the release <a href='https://github.com/jojo2357/Music-Discord-Rich-Presence/releases/tag/1.6.1'>1.6.1</a>.</em>

**Comment by khaledmoayad** [comment]
hey, you were asking for bilingual people to help you add more languages. I can help with arabic if you'd like.

**Reply by jojo2357** [reply]
That would be appreciated. To do that just make a fork of this repo, and then based of the english `.lang` file make a new one named `(whatever arabic is in arabic).lang` and then open a pull request after editing everything that comes after the `==` in the file. If you need help please reach out thru discord and i will be able to assist.
